docs/kintoblocks.md

```diff
@@ -43,6 +43,29 @@ The steps to tag a commit are:
 
 The main reason to tag a KintoBlock is to add it to an application, think of it as a release, so when you have your KintoBlock finished and want to update it in an application you can tag it and select the tag in the application. This is a way to control the released versions of your KintoBlock.
 
-## Custom Parameters (Coming Soon)
+## Custom Parameters
+
+Custom Parameters are another way to pass information to your **KintoApp**. These values are editable by the end user in the case of shared KintoBlocks, and also editable at a KintoApp level for cases where multiple apps are using the same **KintoBlock**.
+
+When creating KintoBlock you can assign default values to your Custom Parameters and these can be marked required if they are necessary for your KintoBlock to function.
```

Comment:
An example of required values would be sendgrid API key and sendgrid api secret.